freeipa/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -35,32 +36,33 @@ public FreeIpaClient(JsonRpcHttpClient jsonRpcHttpClient, String apiVersion) {
         this.apiVersion = apiVersion;
     }
 
-    public RPCResponse<User> userShow(String user) throws FreeIpaClientException {
+    public User userShow(String user) throws FreeIpaClientException {
         List<String> flags = List.of(user);
         Map<String, Object> params = Map.of();
-        return invoke("user_show", flags, params, User.class);
+        return (User) invoke("user_show", flags, params, User.class).getResult();
     }
 
-    public RPCResponse<Set<User>> userFindAll() throws FreeIpaClientException {
+    public Set<User> userFindAll() throws FreeIpaClientException {
         List<String> flags = List.of();
         Map<String, Object> params = Map.of(
                 "sizelimit", 0,
-                "timelimit", 0
+                "timelimit", 0,
```

Comment:
Do we really want to never timeout? Is there a timeout at a higher layer?

Reply:
captured in DISTX-187